project-base/storefront/graphql/docs/schema.md

```
@@ -136,6 +136,7 @@
     * [OrderFilterInput](#orderfilterinput)
     * [OrderInput](#orderinput)
     * [OrderItemsFilterInput](#orderitemsfilterinput)
+    * [OrderWithdrawalRequestInput](#orderwithdrawalrequestinput)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)